style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: #57ca57;
+}
```

Comment:
- [ ] As stated in the project requirement. I would suggest you use the class name given to your h1 tag element in the HTML file to style it in the CSS file. 

E.g :point_down: 
```CSS
.header {
  color: #57ca57
}
```